api/proto/teleport/legacy/types/types.proto

```diff
@@ -6025,3 +6025,135 @@ message GCPMatcher {
   // ProjectIDs are the GCP project ID where the resources are deployed.
   repeated string ProjectIDs = 4 [(gogoproto.jsontag) = "project_ids,omitempty"];
 }
+
+// AccessListV1 describes the basic building block of access grants, which are
+// similar to access requests but for longer lived permissions that need to be
+// regularly audited.
+message AccessListV1 {
+  option (gogoproto.goproto_stringer) = false;
+  option (gogoproto.stringer) = false;
+
+  // Header is the resource header for a resource.
+  ResourceHeader Header = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "",
+    (gogoproto.embed) = true
+  ];
+
+  // Spec is the specification for the access list.
+  AccessListSpecV1 Spec = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "spec"
+  ];
+}
+
+// AccessListSpecV1 is the specification for an access list.
+message AccessListSpecV1 {
+  // Description is a plaintext description of the access list.
+  string Description = 1 [(gogoproto.jsontag) = "description"];
+
+  // Owners is a list of owners of the access list.
+  repeated AccessListOwnerV1 Owners = 2 [(gogoproto.jsontag) = "owners"];
+
+  // Audit describes the frequency that this access list must be audited.
+  AccessListAuditV1 Audit = 3 [(gogoproto.jsontag) = "audit"];
+
+  // MembershipRequires describes the requirements for a user to be a member of the access list.
```

Comment:
Should we consider adding a comment saying that for the access list to be effective, the user must be in members and satisfy `MembershipRequires`. The same for owners